packages/bff/src/model/domain/errors.ts

```diff
@@ -19,3 +22,27 @@ export function purposeNotFound(purposeId: PurposeId): ApiError<ErrorCodes> {
     title: "Purpose not found",
   });
 }
+
+export function missingClaim(claimName: string): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `Claim ${claimName} has not been passed`,
+    code: "missingClaim",
+    title: "Claim not found",
+  });
+}
+
+export function unknownTenantOrigin(selfcareId: string): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `SelfcareID ${selfcareId} is not inside whitelist or related with IPA`,
+    code: "unknownTenantOrigin",
+    title: "Unknown tenant origin",
+  });
+}
```

Comment:
this error does not fit the error condition (Scala version to blame).
I suggest this change
```suggestion
export function tenantLoginNotAllowed(selfcareId: string): ApiError<ErrorCodes> {
  return new ApiError({
    detail: `Tenant origin is not allowed and SelfcareID ${selfcareId} does not belong to allow list`,
    code: "tenantLoginNotAllowed",
    title: "Tenant login not allowed",
  });
}
```